src/components/apis/details-of-api/ko/runtime/api-details.html

```diff
@@ -6,29 +6,29 @@
     <!-- ko ifnot: working -->
     <div class="flex-item flex-grow animation-fade-in">
         <!-- ko if: api -->
-        <h1>
+        <h1 class="title-text">
             <span data-bind="text: api().displayName"></span>
             <!-- ko if: api().type === 'soap' -->
-            <span class="badge badge-soap">SOAP</span>
+            <span class="badge api-type-badge">SOAP</span>
             <!-- /ko -->
             <!-- ko if: api().type === 'websocket' -->
-            <span class="badge badge-soap">WebSocket</span>
+            <span class="badge api-type-badge">WebSocket</span>
             <!-- /ko -->
             <!-- ko if: api().type === 'graphql' -->
-            <span class="badge badge-soap">GraphQL</span>
+            <span class="badge api-type-badge">GraphQL</span>
             <!-- /ko -->
         </h1>
 
         <div class="form-inline">
             <div class="form-group">
                 <!-- ko if: versionApis().length > 0 -->
                 <label for="apiVersions">API version: </label>
-                <select id="apiVersions" class="form-control"
+                <select id="apiVersions" class="api-versions-combobox"
```

Comment:
If it is possible, I would leave `form-control` class and added a modifier class, like yours `api-versions-combobox`